src/Appwrite/Platform/Tasks/Specs.php

```diff
@@ -108,6 +114,24 @@ public function action(string $version, string $mode, Registry $register): void
                     'description' => '',
                     'in' => 'header',
                 ],
+                'Session' => [
+                    'type' => 'apiKey',
+                    'name' => 'X-Appwrite-Session',
+                    'description' => 'The user session to authenticate with',
+                    'in' => 'header',
+                ],
+                'ForwardedFor' => [
+                    'type' => 'apiKey',
+                    'name' => 'X-Forwarded-For',
+                    'description' => 'The IP address of the client that made the request',
+                    'in' => 'header',
+                ],
+                'ForwardedUserAgent' => [
+                    'type' => 'apiKey',
```

Comment:
the type parameters need to be updated here. I dont think these are apiKeys

Reply:
This field is currently unused in SDK generators. Given these objects are headers only string makes sense.

Therefore I think we should remove it and make it implicitly string